Added second way to re-order fields to manual (#316)
ankitects/anki-manual: src/editing.md

![Fields](media/fields.png)

You can add, remove, or rename fields by clicking the appropriate
buttons.

To change the order in which the fields appear in this dialog
and the add notes dialog, you can use the reposition button, which asks
for the numerical position you want the field to have. So if you want to
change a field to be the new first field, enter "1".

Alternatively you can also drag and drop the field names to re-order them. To do

> that, click the field, and drag it to the desired position. An indicator will

Reviewer: ```suggestion
that, use your mouse or finger to drag the field to the desired position. An indicator will
```

![Fields](media/fields.png)

You can add, remove, or rename fields by clicking the appropriate
buttons.

To change the order in which the fields appear in this dialog
and the add notes dialog, you can use the reposition button, which asks
for the numerical position you want the field to have. So if you want to
change a field to be the new first field, enter "1".

Alternatively you can also drag and drop the field names to re-order them. To do

> that, right-click the field, and while still pressing the right mouse button,

Reviewer 2: In other parts of manual, we simply say drag and drop. Or do something like:

```
right-click the field, and drag it to where you want the field to be
```
